Admin/RouteAdmin.php

```diff
@@ -146,7 +146,7 @@ protected function configureFieldsForDefaults($dynamicDefaults)
             '_controller' => array('_controller', Sf2CompatUtil::getFormTypeName('text'), array('required' => false)),
             '_template' => array('_template', Sf2CompatUtil::getFormTypeName('text'), array('required' => false)),
             'type' => array('type', Sf2CompatUtil::getFormTypeName('cmf_routing_route_type'), array(
-                'empty_value' => '',
+                'placeholder' => '',
```

Comment:
This causes `PHP: 5.3 - COMPOSER_FLAGS="--prefer-lowest"` and `PHP: 5.6 SYMFONY_VERSION=2.3.*` to fail (see https://github.com/symfony/symfony/blob/2.8/UPGRADE-2.6.md#form).

What is the plan for Symfony 2.3 support? Do I have to add a fix for that (in `Sf2CompatUtil`) or will support be dropped anyway?